Log troubleshooting information when InvalidInstanceID errors are found during EC2 discovery (#25257)
gravitational/teleport: docs/pages/server-access/guides/ec2-discovery.mdx

It is likely that you're running an older SSM agent version. Upgrade to SSM agent version 3.1 or greater to resolve.

### `InvalidInstanceId: Instances [[i-123]] not in a valid state for account 456`

> The following problems can cause this exception:

Reviewer: ```suggestion
The following problems can cause this error:
```

> - The discovery node don't have permission to access the managed node.

Reviewer: ```suggestion
- The discovery service doesn't have permission to access the managed node.
```

> - AWS Systems Manager Agent(SSM Agent) isn't running. Verify that SSM Agent is running.

Reviewer: ```suggestion
- AWS Systems Manager Agent (SSM Agent) isn't running. Verify that SSM Agent is running.
```
